bufit_test.go

```diff
@@ -146,6 +146,28 @@ func TestQuit(t *testing.T) {
 	}
 }
 
+func TestQuitWriter(t *testing.T) {
+	buf := New()
+
+	wait := make(chan struct{})
+	go func() {
+		for {
+			_, err := io.WriteString(buf, ".")
+			if err != nil {
+				break
+			}
+		}
+		close(wait)
+	}()
```

Comment:
We could drop this goroutine and just call io.WriteString directly after calling buf.Close().